cmd/telliot/main.go

```diff
@@ -132,280 +140,300 @@ func AddDBToCtx(remote bool) error {
 	return nil
 }
 
-var GitTag string
-var GitHash string
+// var GitTag string
+// var GitHash string
 
-const versionMessage = `
-    The official Tellor cli tool %s (%s)
-    -----------------------------------------
-	Website: https://tellor.io
-	Github:  https://github.com/tellor-io/telliot
-`
+// const versionMessage = `
+//     The official Tellor cli tool %s (%s)
+//     -----------------------------------------
+// 	Website: https://tellor.io
+// 	Github:  https://github.com/tellor-io/telliot
+// `
 
-func App() *cli.Cli {
+func (m mineCmd) Run(logger log.Logger) error {
```

Comment:
I would keep the struct functions next to the struct type definition(`cmds.go`). this is idiomatic golang.